project-reports/2024/2024-Q2-Hyperledger-Web3j.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: default
+title: 2024 Q2 Hyperledger Web3j
+parent: 2024
+grand_parent: Project Updates
+---
+
+# Project Health
+
+- The project is in good health.
+- We did 15 releases for all the Web3j Repos fixing several bugs, issues, vulnerabilities and adding new features. 
+- Upgraded Gradle to 8.7 and updated all the dependencies.
+- Fixed and simplified Web3j release process.
+- We got 2 Mentees involved in Web3j-android project through Hyperledger Mentorship Program.
+- Set up a fortnightly contributors call.
+
+# Questions/Issues for the TOC
+
+None at the moment.
+
+# Releases
+
+We did release several strong releases for all Web3j repos.
+
+- Web3j Core
+  - v4.9.8-hotfix for android (April 18, 2024)
+  - v4.11.3 (May 1, 2024)
+  - v4.12.0 (May 23, 2024)
+  
+- Web3j Gradle Plugin
+  - v4.11.3
+  - v4.11.3-hotfix
+  - v4.12.0
+  
+- Web3j Maven Plugin
+  - v4.11.3
+  
+- Web3j Unit
+  - v4.11.3
+  - v4.12.0
+
+- Web3j EVM
+    - v4.11.3
+    - v4.12.0
+
+- Web3j Sokt
+    - v0.4.0
+
+- Web3j Solidity Gradle Plugin
+    - v0.4.1
+    - v0.5.0
+
+- Web3j CLI
+    - v1.6.0
+
+# Overall Activity in the Past Quarter
+
+- 100+ commits and 30 PRs merged in Web3j core.
+- commits and 34 PRs merged across all other Web3j repositories.
+- Fixed bugs, issues, and vulnerabilities, and updated Java and Kotlin versions.
+- Simplified and fixed the release process of Web3j after migrating to the Hyperledger GitHub organization.
+- Upgraded to Gradle 8.7 and updated all dependencies. [Check Announcement](https://medium.com/web3labs/boost-dev-efficiency-security-with-web3js-gradle-8-7-upgrade-d8435a9fb990).
+- Hyperledger Mentorship Program - Mentees started working on web3j-android. [Check project roadmap](https://wiki.hyperledger.org/display/INTERN/Project+Plan+-+Hyperledger+Web3j%3A+Enhancing+Android+Support+with+Updated+web3j-android+Integration).
+- Set up a fortnightly contributors call. A few people joined the call, but it is not very successful yet.
```

Comment:
Do the maintainers at least join this call regularly?

Reply:
Currently, @gtebrean and I are the only maintainers and regularly hold these meetings.